tools/windows-node-installer/test/e2e/aws_test.go

```diff
@@ -303,7 +356,8 @@ func testInstanceIsAssociatedWithClusterWorkerIAM(t *testing.T) {
 	iamProfile, err := awsProvider.GetIAMWorkerRole(infraID)
 	assert.NoError(t, err, "error getting OpenShift Cluster Worker IAM")
 
-	assert.Equal(t, *iamProfile.Arn, *createdInstance.IamInstanceProfile.Arn, "instance is not associated with worker IAM profile")
+	assert.Equal(t, *iamProfile.Arn, *createdInstance.IamInstanceProfile.Arn, "instance is not associated with "+
+		"worker IAM profile")
```

Comment:
Please remove this change. Please do not  make formatting fixes like this. Like I have mentioned before, please check your commit for such changes before opening it up fore review.